### PR TITLE
Replace Carbon Ads with CodeFund

### PR DESCRIPTION
**Pull request by coderberry**

### CODE REVIEW

File: views/_partials/codefund.pug [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.mt-5.mt-md-0.mb-lg-4
+    #codefund
+    script(async, src='https://codefund.io/scripts/c6a10903-6a7f-435b-92b2-99c9c9ce3379/embed.js?target=codefund&theme=bootstrapcdn')
```
Comment (XhmikosR) [inline views/_partials/codefund.pug]: That won't work for the first visit.

File: config/helmet-csp.js [hunk 2]
```diff
@@ -51,7 +51,8 @@ const CSP = {
         'codefund.io',
         'launchbit.com',
         'www.launchbit.com',
-        'www.ziprecruiter.com'
+        'www.ziprecruiter.com',
+        'd2nhukomolqgak.cloudfront.net',
```
Comment (XhmikosR) [inline config/helmet-csp.js]: While this work, you should consider using a subdomain at some point.

File: views/layout.pug [hunk 3]
```diff
@@ -50,7 +50,7 @@ html(lang='en', prefix='og: http://ogp.me/ns#', itemscope, itemtype='http://sche
                 .col-12.col-md-3
                     .row
                         .col-12
-                            include _partials/carbonads.pug
+                            include _partials/codefund.pug
```
Comment (XhmikosR) [inline views/layout.pug]: The old file needs to be removed.

File: config/helmet-csp.js [hunk 4]
```diff
@@ -15,10 +15,7 @@ const CSP = {
         'platform.twitter.com',
         'cdn.syndication.twimg.com',
         'api.github.com',
-        'cdn.carbonads.com',
-        'srv.carbonads.net',
         'adn.fusionads.net',
```
Comment (XhmikosR) [inline config/helmet-csp.js]: I think we can remove this one too.

Comment (jdorfman) [reply config/helmet-csp.js]: what if we need to revert?

Comment (XhmikosR) [reply config/helmet-csp.js]: We'll revert the whole patch anyway, so I'd rather have this complete.

**Comment by XhmikosR** [comment]
I'll verify tomorrow with a clear head the async situation with a clear head.

**Pull request merged**